src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -5,58 +5,73 @@
 import frc.robot.constants.LogPaths;
 import org.littletonrobotics.junction.Logger;
 
+
 public enum BusChain {
 
 	ROBORIO("rio"),
 	CANIVORE("CANivore");
 
+	private static final double MAX_CAN_UTILIZATION_PERCENT = 0.6;
+	private static final double MAX_RECEIVE_ERRORS = 0;
+	private static final double MAX_TRANSMIT_ERRORS = 0;
+	private static final String LOG_PATH_PREFIX = "Bus/";
+
 	private final String chainName;
+	private final String logPath;
+	private CANBus.CANBusStatus busStatus;
 
 	BusChain(String chainName) {
 		this.chainName = chainName;
+		this.logPath = LOG_PATH_PREFIX + this.chainName + "/";
+		this.busStatus = CANBus.getStatus(getChainName());
```

Comment:
don't do getter, or at least be consistent with the way you access the chain name

Reply:
(do the same everywhere)